src/pages/claim-details/claim-actions.tsx

```diff
@@ -119,7 +119,8 @@ export default function ClaimActions(props: Props) {
           return (
             <div className={styles.actionSection}>
               <p className={styles.mediator} data-testid="status-prefix">
-                <Api3Icon aria-hidden /> API3 Mediators
+                <Api3Icon aria-hidden />
```

Comment:
Mhh, some prettier issue?

Reply:
This was just to get rid of the extra white space (after the icon). It was a bit annoying to deal with in the Cypress tests 👍